app/main.py

```diff
@@ -1,3 +1,46 @@
+from __future__ import annotations
+from functools import total_ordering
+
+
+@total_ordering
```

Comment:
Check the note below(it may slow down your program) This task is to learn how to override methods and which magic methods exist. So let's implement them by yourself ;)

Reply:
<img width="786" alt="image" src="https://github.com/user-attachments/assets/b33d30b2-43d1-415d-b1b5-10ad84b6db31">